source/common/res/features/budget-to-goal/main.js

```diff
@@ -0,0 +1,163 @@
+(function poll() {
+  if (typeof ynabToolKit !== 'undefined' && ynabToolKit.pageReady === true) {
+    ynabToolKit.budgetBalanceToGoal = (function () {
+      return {
+        budgetView: ynab.YNABSharedLib.getBudgetViewModel_AllBudgetMonthsViewModel()._result,
+
+        invoke() {
+          let categories = ynabToolKit.budgetBalanceToGoal.getCategories();
+          let categoryName = ynabToolKit.budgetBalanceToGoal.getInspectorName();
+          let masterCategoryViewId = $('ul.is-checked').prevAll('ul.is-master-category').attr('id');
+          let masterCategory = ynabToolKit.shared.getEmberView(masterCategoryViewId).get('data');
+          let masterCategoryId = masterCategory.get('categoryId');
+
+          categories.forEach(function (f) {
+            if (f.name === categoryName && f.masterCategoryId === masterCategoryId) {
+              ynabToolKit.budgetBalanceToGoal.updateInspectorButton(f);
+
+              return false;
+            }
+          });
+        },
+
+        observe(changedNodes) {
+          console.log(changedNodes);
```

Comment:
You'll definitely want to remove this log statement before we merge, as this'd slow things down quite a bit.